src/styles/team.scss

```diff
@@ -0,0 +1,39 @@
+.test {
+  background-color: white;
+}
+
+.teams {
+  background-color: $dark-blue;
+  color: black;
+
+  &__button {
+    margin-left: 10px;
+    margin-bottom: 10px
+  }
+
+  .selected {
+    color: white;
+    background-color: $orange;
+    border-width: 0;
+    box-shadow: none;
+  }
+  
+  &__card {
```

Comment:
is there global card styling that we are using on all cards throughout our site? wondering if these & mentors are going to be the same or if it makes sense to have their styling separate

Reply:
Oh thank you! I had meant to make one central card styling definition, but I forgot